packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -93,13 +93,23 @@ module.exports = {
     }
   },
 
-  // Truffle DB is currently disabled by default; to enable it, change enabled: false to enabled: true
+  // Truffle DB is currently disabled by default; to enable it, change enabled:
+  // false to enabled: true. The default storage location can also be
+  // overridden by specifying the adapter settings, as shown in the commented code below.
   //
-  // Note: if you migrated your contracts prior to enabling this field in your Truffle project and want
-  // those previously migrated contracts available in the .db directory, you will need to run the following:
+  // NOTE: It is not possible to migrate your contracts to truffle DB and you should
+  // make a backup of your artifacts to a safe location before enabling this feature.
+  //
+  // After you backed up your artifacts you can utilize db by running migrate as follows: 
   // $ truffle migrate --reset --compile-all
 
   db: {
-    enabled: false
+    enabled: false,
```

Comment:
do we need to comment this out to be consistent with the metacoin box?

Reply:
Or alternatively uncomment it there I suppose. I don't have a preference, just want consistency.